Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/EventCheckIn.kt

```diff
@@ -6,6 +6,16 @@ interface EventCheckIn {
 
     val id: Long
     val guid: String
-    val startTime: Instant
-    val endTime: Instant
+    val version: Int
+    val type: Int
+    val description: String
+    val address: String
+    val traceLocationStart: Instant
+    val traceLocationEnd: Instant
+    val defaultCheckInLengthInMinutes: Int
+    val signature: String
+    val checkInStart: Instant
+    val checkInEnd: Instant
```

Comment:
`checkInEnd` and `targetCheckInEnd` can also be null

Reply:
fixed it!